devops/jobs/ToolsGpUsers.groovy

```diff
@@ -0,0 +1,111 @@
+/*
+
+ Variables without defaults are marked (required) 
+ 
+ Variables consumed for this job:
+    * SECURE_GIT_CREDENTIALS: secure-bot-user (required)
+    * NOTIFY_ON_FAILURE: alert@example.com
+    * FOLER_NAME: folder, default is User-Mananagement
+    * ACCESS_CONTROL: list of users to give access to
+        - user
+    * CONFIGURATION_SECURE_REPO: repository where ansible vars are located (required)
+ 
+ This job expects the following credentials to be defined on the folder
+    ubuntu_deployment_201407: ssh key
```

Comment:
Can we pass in the expected name of this credentials via secure config as well.  At some point in the future we'll want to add the new credential and switch to it before deleting the old one and we'll want the name to reflect the content.